test/Parse/trailing_closure.swift

```diff
@@ -0,0 +1,10 @@
+// RUN: %target-typecheck-verify-swift
+
+func f()->Int{ 42 }
+
+//This should be interpreted as a trailing closure, instead of being interpreted as a computed property with undesired initial value.
```

Comment:
Nit: Please try to keep the spacing and line widths consistent with the rest of the project. This means that comments should have a space after the `//`, and they need to be wrapped to 80 columns.

For functions, the arrow should have spaces on both sides (and similarly, there needs to be a space before an opening brace).

```
func f()->Int{ 42 } // right now
func f() -> Int { 42 } // match existing style
```

Reply:
Thanks for your kind instruction. I've made the changes. A small question--when will I be qualified to trigger CI?

Reply:
There is a commit access section on https://swift.org/contributing/#contributing-code which describes the details. Usually commit access and swift-ci access are granted together.